student-course-registration-system

```diff
@@ -0,0 +1 @@
+Subproject commit 6d0cbfc0da406c6fcad93f6fb37c9fb7e3a3feac
```

Comment:
What is this?

Reply:
No idea what is this actually. I swear I didn't add this. I tried to search this in VS Code, but no result. This seems to be like didn't add into any of the files there.